app/tests/test_queries.py

```diff
@@ -1539,44 +2192,220 @@ def test_browse_transferring_body_with_records_held_sorting_most_first(
+                "15/03/2023",
+                3,
+                browse_files[3].consignment.ConsignmentId,
+                browse_files[3].consignment.ConsignmentReference,
+            ),
+        ]
+
+        results = pagination_object.items
 
-        file = files[0]
-        mock_standard_user(client, file.consignment.series.body.Name)
+        assert results == expected_results
 
-        transferring_body_id = file.consignment.series.body.BodyId
-        series_id = file.consignment.series.SeriesId
-        consignment_id = file.consignment.ConsignmentId
+    def test_browse_series_with_date_consignment_transferred_filter_using_date_from_and_date_to_option(
+        self, client: FlaskClient, mock_standard_user, browse_files
+    ):
+        """
+        Given 27 file objects with all file type as 'file'
+        And the session contains user info for a standard user with access to the consignment's
+            associated transferring body
+        When I call the 'browse_data' function with browse_view as 'series' and
+            'date from' and 'date to' as filter
+        Then it returns a Paginate object returning the first 1 item
+            for a specific series
+            consignment transfer complete date which is greater than or equal to date from
+            and less than or equal to the date to value
+        """
+        mock_standard_user(client, browse_files[6].consignment.series.body.Name)
+        series_id = browse_files[6].consignment.series.SeriesId
 
-        result = browse_data(
-            page=1, per_page=per_page, browse_type="series", series_id=series_id
+        filters = {
+            "date_range": {"date_from": "01/03/2023", "date_to": "30/03/2023"}
+        }
+        pagination_object = browse_data(
+            page=1,
+            per_page=per_page,
+            browse_type="series",
+            series_id=series_id,
+            filters=filters,
         )
 
-        assert result.items == [
+        assert pagination_object.total == 1
+
+        expected_results = [
             (
-                transferring_body_id,
-                "test body1",
-                series_id,
-                "test series1",
-                "01/01/2023",
-                1,
-                consignment_id,
-                "test consignment1",
-            )
+                browse_files[3].consignment.series.body.BodyId,
+                browse_files[3].consignment.series.body.Name,
+                browse_files[3].consignment.series.SeriesId,
+                browse_files[3].consignment.series.Name,
+                "15/03/2023",
+                3,
+                browse_files[3].consignment.ConsignmentId,
+                browse_files[3].consignment.ConsignmentReference,
+            ),
         ]
 
+        results = pagination_object.items
+
+        assert results == expected_results
+
+    def test_browse_series_with_date_consignment_transferred_filter_using_date_from_and_date_to_option_no_result(
```

Comment:
This file is even huuger than before. We should start breaking out test_queries.py into one test file per query function at minimum

Reply:
Like mentioned before, we would be able to reduce the amount of tests and cover more permutations more efficiently by testing various edge cases at a lower unit test level and then having one or two integration tests per route route / query and a function composition test to show that queries are built up by main query, filter and sort. 
We have a ticket for that but just wanted to raise it again so we keep it in mind

Reply:
I agree, need to split into four views test cases separately,